css/properties/text-underline-position.json

```diff
@@ -59,7 +59,7 @@
                 "version_added": "33"
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": false
```

Comment:
My information says Chrome on Android supports this. If you've found otherwise I may need to file a Chrome bug.